pkg/cache/application_events.go

```diff
@@ -160,3 +162,34 @@ func (fe FailApplicationEvent) GetArgs() []interface{} {
 func (fe FailApplicationEvent) GetApplicationID() string {
 	return fe.applicationID
 }
+
+// ------------------------
+// Release application
+// ------------------------
+type ReleaseApplicationEvent struct {
+	applicationID  string
+	allocationUUID string
+	event          events.ApplicationEventType
+}
```

Comment:
can you pls add the TERMINATION_TYPE in this message as well?
this gives us the flexibility in case we want to handle different types differently in the future

Reply:
OK, done